tools/Google.Cloud.Tools.VersionCompat.Tests/ClassMethodTest.cs

```diff
@@ -63,5 +97,19 @@ public class ClassMethodTest : TestBase
         [Fact] public void ParamInOutChanged2() => TestMajor(Cause.MethodParameterInOutChanged);
         [Fact] public void ToStatic() => TestMajor(Cause.MethodMadeStatic);
         [Fact] public void FromStatic() => TestMajor(Cause.MethodMadeNonStatic);
+        [Fact] public void GenericConstraintAdded1() => TestMajor(Cause.MethodGenericConstraintChanged);
+        [Fact] public void GenericConstraintAdded2() => TestMajor(Cause.MethodGenericConstraintChanged);
+        [Fact] public void GenericConstraintAdded3() => TestMajor(Cause.MethodGenericConstraintChanged);
+        [Fact] public void GenericConstraintAdded4() => TestMajor(Cause.MethodGenericConstraintChanged);
+        [Fact] public void GenericConstraintRemoved1() => TestMinor(Cause.MethodGenericConstraintChanged);
```

Comment:
I'm not convinced that this isn't a breaking change, due to overload resolution changes in C# 7.3.
Generic constraints are now included in applicability tests, so I wonder whether removing a constraint *may* make something ambiguous.

Would you like me to try to work up an example?

Reply:
If you've got time, then that'd be helpful, thanks :)